net-libs/libsrtp/libsrtp-1.5.4.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="Open-source implementation of the Secure Real-time Transport Protocol (SRTP)"
+HOMEPAGE="https://github.com/cisco/libsrtp"
+SRC_URI="https://github.com/cisco/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/1"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 -sparc ~x86 ~x86-fbsd ~ppc-macos ~x64-macos ~x86-macos"
+IUSE="aesicm console debug doc libressl openssl static-libs syslog test"
+
+DEPEND="
+	openssl? ( dev-libs/openssl:0= )
+	libressl? ( dev-libs/libressl:0= )
+"
```

Comment:
@lluixhi I will repeat it one last time. `USE="libressl"` does **NOT** mean "use libressl". What `USE="libressl"` means, is "use libressl, **if and only if**, building against an openssl backend". If you do not follow this line of reasoning, see the project page https://wiki.gentoo.org/wiki/Project:LibreSSL which just reiterates all of this. The correct solution is:
```
openssl? (
    !libressl? ( dev-libs/openssl:0= )
    libressl? ( dev-libs/libressl:0= )
)
```
i.e. `USE="openssl libressl"` -> _libressl_, `USE="openssl -libressl"` -> _openssl_, `USE="-openssl"` -> _no SSL_. If you won't change it to this, I will close the PR for not being in line with the libressl project guidelines.